config/nrfconnect/.nrfconnect-recommended-revision

```diff
@@ -1 +1 @@
-ffcf07fe4586634a6793a48e5444a7196e7ebac6
+cfedfdfa08567b2252b511a4d1db15fbeba8152d
```

Comment:
Please do this in a separate PR when we update all CI to use the latest VC version.

We have builds that only do `--check` (because of regular user vscode permission) and we want to keep CI and this recommented revision in sync.

Reply:
I can do the opposite and extract the Docker changes to a separate PR. This change must be aligned with changes it the code.

Reply:
Why exactly is it impossible to update the NCS in the VS Code btw. Normally, you are allowed to modify files provided by an image.

Reply:
https://github.com/project-chip/connectedhomeip/pull/13034

Reply:
PR merged, I am building the 5.40 images now (usually takes about 1hr).

Reply:
for the permissions, we used to not have sufficient chmod for the vscode user, but that was fixed last week I believe. We still retain the --check logic and no update though, with the rationale of "running a compile would not generally be expected to update SDK installations".

Reply:
0.5.40 for nrf was updloaded to dockerhub.

Reply:
Thanks, Andy :)